### PR TITLE
Update etracker source URL

### PR DESCRIPTION
**Pull request by ben-dw**
Changed source url of etracker code + added async attribut

### Description:

As etracker has updated it URL code, i have changed it.

### Review

* [x] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [x] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [x] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [x] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [x] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [x] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by snake14 — APPROVED** [review]
Looks good to me and it seemed to load the etracker JS when I tested it in my Matomo install. Any concerns @AltamashShaikh ?

File: Template/Tag/EtrackerTag.web.js [hunk 1]
```diff
@@ -44,7 +44,8 @@
                 if (etrackerConfig.etrackerDNT) {
                     script.setAttribute('data-respect-dnt', 'true');
                 }
-                script.src = '//static.etracker.com/code/e.js';
+                script.src = '//code.etracker.com/code/e.js';
+                script.setAttribute('async', '');
```
Comment (AltamashShaikh) [inline Template/Tag/EtrackerTag.web.js:48]: @ben-dw If we change to async, we might have to adjust the `sendEvent` code too as per this [link](https://www.etracker.com/en/docs/faq-2/core-web-vitals/how-do-i-adjust-the-etracker-code-so-that-the-etracker-script-is-loaded-asynchronously/)

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
@ben-dw Left 1 comment for async changes, rest looks good

File: Template/Tag/EtrackerTag.web.js (hunk 1 above)
Comment (ben-dw) [reply Template/Tag/EtrackerTag.web.js:48]: @AltamashShaikh : Only ecommerce events have to be changed. I will double check within #311

Comment (ben-dw) [reply Template/Tag/EtrackerTag.web.js:48]: I have changed it in #311

**Review by AltamashShaikh — APPROVED** [review]

**Pull request merged**